nautobot/extras/tables.py

```diff
@@ -527,10 +524,16 @@ def related_object_link(value, record):
     """
```

Comment:
Diffs still show `value` as a parameter to `related_object_link` - shouldn't that be removed?

Reply:
I suspect this may have accidentally been reverted during the merge resolution when the function name changed.